Products/ZenUI3/browser/javascript.py

```diff
@@ -251,13 +251,19 @@ def snippet(self):
         # 3. the timezone of the browser
         user = self.context.dmd.ZenUsers.getUserSettings()
         timezone = user.timezone
+        date_fmt = user.dateFormat
         snippet = """
+          (function(){
+            Ext.namespace('Zenoss.env');
+
             Zenoss.env.COLLECTORS = %r;
             Zenoss.env.priorities = %r;
             Zenoss.env.productionStates = %r;
             Zenoss.USER_TIMEZONE = "%s" || jstz.determine().name();
-        """ % ( collectors, priorities, productionStates, timezone )
-        return snippet
+            Zenoss.USER_DATE_FORMAT = "%s" || "MM/DD/YY";
+          })();
+        """ % ( collectors, priorities, productionStates, timezone, date_fmt )
+        return SCRIPT_TAG_TEMPLATE % snippet
```

Comment:
This change is probably not what is expected. This function used to return a snippet of javascript, but this change to insert the snippet into `SCRIPT_TAG_TEMPLATE` makes this function return a full `<script>` tag with the javascript snippet inside. I would avoid changing how it works unless you're sure that it makes sense.